ibis/sql/sqlite/client.py

```diff
@@ -91,15 +93,22 @@ def table(self, name, database=None):
         Parameters
         ----------
         name : string
+        database : string, optional
+          name of the attached database that the table is located in.
 
         Returns
         -------
         table : TableExpr
         """
-        alch_table = self._get_sqla_table(name)
+        alch_table = self._get_sqla_table(name, schema=database)
         node = SQLiteTable(alch_table, self)
         return self._table_expr_klass(node)
 
+    def list_tables(self, like=None, database=None, schema=None):
+        if database is None:
+            database = self.database_name
+        return super(SQLiteClient, self).list_tables(like, schema=database)
```

Comment:
So for the sqlite things it's a terminology thing again. 
SQLAlchemy treats each attached database as a schema